### ISSUE TITLE
Arbitrary Critical Error

### ISSUE THREAD

**Issue by VehementHam** [nicotine-plus/nicotine-plus#2996]
Nicotine+ version:  `Nicotine+ 3.3.2`
Operating System/Distribution: Alpine Linux Edge

### Describe the bug
I recieved a "Critical Error" messaget that advised me to create a bug report. Here is the error:
```
Nicotine+ Version: 3.3.2
GTK Version: 3.24.41
Python Version: 3.12.3 (linux)

Type: <class 'TypeError'>
Value: '>' not supported between instances of 'int' and 'NoneType'
Traceback:   File "/usr/lib/python3.12/site-packages/pynicotine/gtkgui/widgets/iconnotebook.py", line 621, in on_focus_page
    if not page.focus_callback():
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/pynicotine/gtkgui/transfers.py", line 264, in on_focus
    self.update_model()
  File "/usr/lib/python3.12/site-packages/pynicotine/gtkgui/transfers.py", line 380, in update_model
    row_added = self.update_specific(transfer_i, select_parent, use_reverse_file_path)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/pynicotine/gtkgui/transfers.py", line 748, in update_specific
    self.get_percent(current_byte_offset, size),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/pynicotine/gtkgui/transfers.py", line 462, in get_percent
    if current_byte_offset > size or size <= 0:
       ^^^^^^^^^^^^^^^^^^^^^^^^^^
```

### Expected behavior
For Nicotine not to experience critical errors.

Sorry for the crappy bug report I have no idea what caused this "critical error," and I have not been expereinceing any other noticable issues.

**Comment by mathiascode** [comment]
Duplicate of #2953

**Comment by VehementHam** [comment]
I think I just moved my shares direcotry, so it didn't exist anymore.

**Comment by mathiascode** [comment]
Yep, that's what caused the crash. It's fixed in 3.3.3, which is rolling out now.
